src/Components/WebAssembly/testassets/WasmPrerendered.Client/WasmPrerendered.Client.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk.BlazorWebAssembly">
+
+  <PropertyGroup>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Components.WebAssembly" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="wwwroot\" />
+  </ItemGroup>
+
```

Comment:
I think this block might not be required, but maybe there's some subtlety I'm missing. Also it doesn't matter, so feel free to leave it if you have more important things to focus on!

Reply:
Also on the subject of not-very-important feedback, if the names of these folders/projects had an extra `.` in them (e.g., `Wasm.Prerendered.Client/Wasm.Prerendered.Client.csproj `) that would align with the existing `Wasm.Authentication.*` folders/projects.

Again, this is not important so please only react to it if you really want to!